src/components/interfaces/MOBILE_API.xml

```diff
@@ -3161,11 +3167,20 @@
                 True: Available, False: Not Available, Not present: Not Available.
             </description>
         </param>
-        <param name="availableHDsAvailable" type="Boolean" mandatory="false">
+        <param name="availableHDsAvailable" type="Boolean" mandatory="false" deprecated="true" since="5.1">
             <description>
                 Availability of the getting the number of available HD channels.
                 True: Available, False: Not Available, Not present: Not Available.
             </description>
+            <history>
+                <param name="availableHDsAvailable" type="Boolean" mandatory="false" since="1.0" until="5.1"/>
```

Comment:
@IGapchuk do you really need "since" here?

Reply:
@IGapchuk I believe, "until=5.0"

Reply:
@AKalinich-Luxoft Yes, we need "since" here, because elements 'history' doesn't work without "since" and only with "untill".

Reply:
@mkorniichuk No, we have to point the version, beginning from what this parameter stops work.